be/src/storage/delta_writer.cpp

```diff
@@ -515,7 +518,7 @@ void DeltaWriter::abort(bool with_log) {
         _flush_token->shutdown();
     }
     if (_replicate_token != nullptr) {
```

Comment:
Why do not call cancel() in the abort()?

Reply:
`Cancel` is called in `LoadChannelMgr`

```
        if (auto channel = remove_load_channel(load_id); channel != nullptr) {
            channel->cancel();
            channel->abort();
        }
```

So, no need to invoke here